packages/devtools_app/lib/src/app_size/app_size_screen.dart

```diff
@@ -35,13 +35,19 @@ class AppSizeScreen extends Screen {
 
   static const analysisTabKey = Key('Analysis Tab');
   static const diffTabKey = Key('Diff Tab');
-  static const id = 'app-size';
+
+  /// The ID (used in routing) for the tabbed app-size page.
+  ///
+  /// This must be different to the top-level appSizePageId which is also used
+  /// in routing when to ensure they have unique URLs.
+  static const id = 'appsize';
```

Comment:
does this break the documentation link (bottom right of devtools)? If so we should change the other id

Reply:
Yeah, it did :(

I've swapped them around (another option would've been to set `docId`, though this seemed simpler). Thanks!